src/components/PageBlocks/Configurator.vue

```diff
@@ -80,14 +80,36 @@
           />
         </b-form-group>
 
-        <b-form-checkbox
-          v-model="block.style.wrap.kind"
-          value="card"
-          unchecked-value="plain"
-          switch
+        <b-form-group>
+          <b-form-checkbox
+            v-model="block.style.wrap.kind"
+            value="card"
+            unchecked-value="plain"
+            switch
+          >
+            {{ $t('general.wrap') }}
+          </b-form-checkbox>
+        </b-form-group>
+        <b-form-group
+          :label="$t('general.refresh.label')"
+          :description="$t('general.refresh.description')"
         >
-          {{ $t('general.wrap') }}
-        </b-form-checkbox>
+          <b-input-group>
+            <b-form-input
+              v-model="block.options.refreshRate"
+              class="col-1"
+              type="number"
+              number
+              min="0"
+              @blur="updateRefresh"
+            />
+            <b-input-group-append>
```

Comment:
Instead of doing this, just use the append property on the b-input-group

Reply:
It sorts of add background to it that is why I opted for this according to the demo in the docs.
![image](https://user-images.githubusercontent.com/31524469/200053434-4b6fa7e4-39d2-47cb-bd2f-2a149e0d5381.png)